### ISSUE TITLE
default depth threshold has changed from 50 to 20?

### ISSUE THREAD

**Issue by cathrnbp** [polio-nanopore/piranha#260]
Using the command line piranha, consensus sequences are being called in the detailed_run_report despite having less than 50 reads. Minimum now seems to be 20? Can the default be changed back to 50?

**Comment by s-mobed** [comment]
Hello, 
wondering if it is still possible to set the minimum depth back to 50 in the config file, as well as the minimum read percent to 0

**Comment by ZoeVance** [comment]
Yes, min read depth can be set to 50 and min read percent to 0 by adding

min_read_depth: 50
min_read_pcent: 0

to the config. 

Probably worth mentioning given discussions around Ns in consensus sequences last week that increasing the min read depth option will affect masking in samples with lower numbers of reads. For a site not to be masked it needs min read depth coverage at that site i.e. that number of reads that both cover that site and have that base passing the quality cutoff. For samples where the overall number of mapping reads is close to the depth cutoff many individual positions won't meet it because not every read covers every position, not every base passes the quality cutoff. This is the case with the samples that were discussed with high (or all) Ns in the consensus last week. For those samples piranha returns much more complete consensus sequences with the default min 20 reads, so maybe something that needs to be considered.

**Comment by s-mobed** [comment]
Those aren't the default settings right now; it's min_depth:20 and min_pcent 2. I was wondering why they changed ? That was more my question. Would it be possible to set them back to a depth of 50 and a percentage of 0? Since these are the required settings outlined in the PoSeCo protocols.

I have a question about the masking, what is the quality cutoff you use? and is there a flag that we can use to set that threshold ?

**Comment by aineniamh** [comment]
Hi, this has been discussed in the polio meetings- the default changed some time ago (over a year) after medaka became better at calling consensus sequences. More recently we discussed this change and it was decided to stick with the 50 minimum. I offered to change it back in piranha, and I was instructed to not make any changes to piranha re this, so it's still at the 20 depth by default. I'm on leave but please bring it up in the meetings and I'd be happy to change it back if given the go-ahead

**Comment by ZoeVance** [comment]
> I have a question about the masking, what is the quality cutoff you use? and is there a flag that we can use to set that threshold ?

The cutoff is a base quality of 13; I believe there's a minimum mapping quality flag in piranha but not one for base quality. We could probably implement user setting of that threshold if it would be useful to be able to increase it, but I don't think lowering it would be advisable. 13 is a pretty standard cutoff and is relatively lax; I personally don't really consider a base trustworthy below q=20.

If you have more questions/requests around the depth masking could you open a new issue? Just to keep it clear from open issues what has/hasn't been resolved, this feels separate to the question of setting defaults.

**Comment by aineniamh** [comment]
latest release reverts to the following

```
STOOL DEFAULT
min_reads = 50
min_pcent = 0

ENVIRONMENTAL DEFAULT
min_reads = 20
min_pcent = 0

ISOLATE DEFAULT
min_reads = 500
min_pcent = 0
```
